src/main/java/mate/academy/internetshop/controller/order/OrdersAllController.java

```diff
@@ -9,16 +9,21 @@
 import mate.academy.internetshop.lib.Injector;
 import mate.academy.internetshop.model.Order;
 import mate.academy.internetshop.service.OrderService;
+import mate.academy.internetshop.service.UserService;
 
 public class OrdersAllController extends HttpServlet {
```

Comment:
```suggestion
public class GetUserOrdersController extends HttpServlet {
```
 ми ж тут витягуємо усі ордери юзера а не усі існуючі, думаю так буде коректніше

Reply:
Done